pysat/tests/test_meta.py

```diff
@@ -691,36 +702,38 @@ def test_replace_meta_units_list(self):
         assert self.meta['new2'].units == 'yeppers'
         assert self.meta['new2'].long_name == 'boo2'
 
-    def test_meta_repr_functions(self):
-        self.testInst.meta['new'] = {'units': 'hey', 'long_name': 'boo'}
-        self.testInst.meta['new2'] = {'units': 'hey2', 'long_name': 'boo2'}
-        print(self.testInst.meta)
-        # if it doesn't produce an error, we presume it works
-        # how do you test a print??
-        assert True
-
```

Comment:
Removed because I don't see what this adds compared to the above functions.  I think duplicate tests may have been written at different points.